From a92e2347531453af60510b0ce689d013568fa381 Mon Sep 17 00:00:00 2001
From: John Crispin <john@phrozen.org>
Date: Wed, 1 Oct 2025 08:29:35 +0200
Subject: [PATCH] kernel: add MediaTek SDK kernel 5.4.271 hash

Add kernel version and hash for MediaTek target using Linux 5.4.271
from the MediaTek SDK external feed.

Signed-off-by: John Crispin <john@phrozen.org>
---
 include/kernel-5.4 | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/include/kernel-5.4 b/include/kernel-5.4
index b5ae3eaa28..0d9c654dd8 100644
--- a/include/kernel-5.4
+++ b/include/kernel-5.4
@@ -5,3 +5,8 @@ ifeq ($(CONFIG_TARGET_ipq95xx)$(CONFIG_TARGET_ipq53xx),y)
 LINUX_VERSION-5.4 = .213
 LINUX_KERNEL_HASH-5.4.213 = 53d191b59994a3b8b5c9cac7391cbf0a4186ba4bd08b9f378e2d05bfd4921f6d
 endif
+
+ifeq ($(CONFIG_TARGET_mediatek),y)
+LINUX_VERSION-5.4 = .271
+LINUX_KERNEL_HASH-5.4.271 = cdbc61334cdadbd3945b08f03ed197c65bdf358c3383a4334b3e5b483bd95850
+endif
-- 
2.34.1

